src/ngrest/plugins/Injector.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace luya\admin\ngrest\plugins;
+
+use luya\admin\ngrest\base\Plugin;
+use luya\helpers\Html as HtmlHelper;
+
+/**
+ * Create a text input select for a given field.
+ *
+ * @author Bennet Klarhoelter <boehsermoe@me.com>
+ * @since 1.2.3
+ */
+class Injector extends Plugin
+{
+    /**
+     * @var string
+     */
+    public $dir;
```

Comment:
What does $dir means?

Reply:
the zaa-directive like there https://github.com/luyadev/luya-module-admin/blob/master/src/resources/js/directives.js#L719